### ISSUE TITLE
Prepare docs!

### ISSUE THREAD

**Issue by mertbozkir** [mertbozkir/DVC.jl#3]
- [ ] Prepare docs and instructions about how to use DVC.jl

**Comment by adityakode** [comment]
@mertbozkir Would love to help!
Can I work on this issue?

**Comment by mertbozkir** [comment]
Yeah of course! Give me sometime to plan how it would be effective for deploying our docs.

**Comment by adityakode** [comment]
@mertbozkir  sure!
